src/plugins/telemetry/public/plugin.ts

```diff
@@ -87,6 +88,7 @@ export interface TelemetryPluginStart {
 interface TelemetryPluginSetupDependencies {
   screenshotMode: ScreenshotModePluginSetup;
   home?: HomePublicPluginSetup;
+  cloud?: CloudSetup;
```

Comment:
I think we are missing the changes in the `kibana.json` manifest to declare `cloud` as an optional dependency